apache/superset
# chore: Remove Control.less in Explore

### SUMMARY
It removes the Control.less file from the Control component in Explore.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
No visual changes

### TESTING INSTRUCTIONS
1. Open Explore
2. Make sure there are no visual changes

### ADDITIONAL INFORMATION
